akka-typed-tests/src/test/scala/akka/typed/TypedSpec.scala

```diff
@@ -202,13 +202,15 @@ class TypedSpecSpec extends TypedSpec {
   object `A TypedSpec` {
 
     trait CommonTests {
+      class MyException(message: String) extends Exception(message)
+
       implicit def system: ActorSystem[TypedSpec.Command]
 
       def `must report failures`(): Unit = {
-        a[TestFailedException] must be thrownBy {
+        a[MyException] must be thrownBy {
           sync(runTest("failure")(StepWise[String]((ctx, startWith) ⇒
             startWith {
-              fail("expected")
+              throw new MyException("expected")
```

Comment:
Oh, sorry, it's a prepared test exception, but it's only in the stream teskit I see now: https://github.com/akka/akka/blob/23ca48bdcca096170db3c396b762ff8dd3a91737/akka-stream-testkit/src/test/scala/akka/stream/testkit/Utils.scala